hardhat.config.ts

```diff
@@ -0,0 +1,199 @@
+    governanceZombie: "0.03",
+  },
+  maxFees: {
+    curve: "0.01",
+    flat: "0.001",
+    governanceLP: "0.15",
+    governanceZombie: "0.03",
+  },
+};
+
+const TEST_ERC4626: ERC4626InstanceDeployConfigInput = {
+  name: "TEST_ERC4626",
+  deploymentId: "0x666",
+  salt: "0x694201",
+  contribution: "0.1",
+  fixedAPR: "0.5",
+  timestretchAPR: "0.5",
+  options: {
+    // destination: "0xsomeone",
+    asBase: true,
+    // extraData: "0x",
+  },
+  poolDeployConfig: {
+    // baseToken: "0x...",
+    // vaultSharesToken: "0x...",
+    minimumShareReserves: "0.001",
+    minimumTransactionAmount: "0.001",
+    positionDuration: "30 days",
+    checkpointDuration: "1 day",
+    timeStretch: "0",
+    governance: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
+    feeCollector: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
+    sweepCollector: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
+    fees: {
+      curve: "0.001",
+      flat: "0.0001",
+      governanceLP: "0.15",
+      governanceZombie: "0.03",
+    },
+  },
+};
+
+const TEST_STETH: StETHInstanceDeployConfigInput = {
+  name: "TEST_STETH",
+  deploymentId: "0x666666",
+  salt: "0x69420",
+  contribution: "0.1",
+  fixedAPR: "0.5",
+  timestretchAPR: "0.5",
+  options: {
+    // destination: "0xsomeone",
+    asBase: false,
+    // extraData: "0x",
+  },
+  poolDeployConfig: {
+    // vaultSharesToken: "0x...",
+    minimumShareReserves: "0.001",
+    minimumTransactionAmount: "0.001",
+    positionDuration: "30 days",
+    checkpointDuration: "1 day",
+    timeStretch: "0",
+    governance: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
+    feeCollector: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
+    sweepCollector: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
+    fees: {
+      curve: "0.001",
+      flat: "0.0001",
+      governanceLP: "0.15",
+      governanceZombie: "0.03",
+    },
+  },
+};
+
+const TEST_RETH: RETHInstanceDeployConfigInput = {
+  name: "TEST_RETH",
+  deploymentId: "0x666666666",
+  salt: "0x69420",
+  contribution: "0.1",
+  fixedAPR: "0.5",
+  timestretchAPR: "0.5",
+  options: {
+    // destination: "0xsomeone",
+    asBase: false,
+    // extraData: "0x",
+  },
+  poolDeployConfig: {
+    // vaultSharesToken: "0x...",
+    minimumShareReserves: "0.001",
+    minimumTransactionAmount: "0.001",
+    positionDuration: "30 days",
+    checkpointDuration: "1 day",
+    timeStretch: "0",
+    governance: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
+    feeCollector: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
+    sweepCollector: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
+    fees: {
+      curve: "0.001",
+      flat: "0.0001",
+      governanceLP: "0.15",
+      governanceZombie: "0.03",
+    },
+  },
+};
+
+const { env } = process;
+const config: HardhatUserConfig = {
+  solidity: {
+    version: "0.8.20",
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 10_000_000,
+      },
+      evmVersion: "paris",
```

Comment:
We should probably switch to `cancun`. We should probably do this in foundry too FWIW